dev-libs/libclc/Manifest

```diff
@@ -2,3 +2,4 @@ DIST libclc-0.2.0_pre20160921.tar.gz 127817 BLAKE2B 831bdc4a82d50ec33eb3199ef707
 DIST libclc-0.2.0_pre20170118.tar.gz 130793 BLAKE2B 5083e821c5a0e481d3b6b638ac93c8c936972dcdb78e20a86de5f6873f300d7065e8aba242b332d4a91d494bb3a9e67f1f60a566bfbef78891d7c438bebbf41c SHA512 d94a58ebcafb7bbdbcded84574676537ae7b4ba86c7c2270cc1cd7aa3113104e018e53396b570e821c2fd473b6dd63677f59d990e36e15a7cdb43cdb96cd058c
 DIST libclc-0.2.0_pre20170929.tar.gz 138719 BLAKE2B 05a3d47626a5e1918f32510b1254f846c88c642f60920377dda73568d34656fecd53872ebc01114b2cc8b2161b196c89efe119bfd785875f723fcee4c1ce3662 SHA512 08f3a21fb1dc103a3b33980c420506fd142294b2d8b4a3eeb100ea55036fb87b9740a2489a0a9a5793d71ebb157e6f3ad7854d1916fc1fa472de72374cdeaea3
 DIST libclc-0.2.0_pre20180319.tar.gz 161608 BLAKE2B b8e4f2d475c0425cbe5b41a5ba3b7b40d14c045f5c23d007e87cdfedc7a01a8317218e5f3b881e944256f04b9fb68a130bff2a137e51cb961b3d5bb7f2e2261d SHA512 5a4130eeb416f90901b66708adc17c825baa757d00d983a12e217f36503e4ed5d208f3a49cc26d7f9b71134ee3d258748153bf52cf7769bcbc06ec0510087a36
+DIST libclc-0.2.0_pre20180610.tar.gz 170102 BLAKE2B 3f8c33a9961db34e68cebc328a88a71173000972b5540d099fa79fe89fd186b7f2f8791a86b45b1be135383d26a92fd4ccdc2827b5620b5f86f0b48467fc092f SHA512 0c75c3e7b81754c175c5c43614ad313cda2fa09a1865794c904675b8d2527f78ffa357178f7d4a471b52ac3d7d96b0184303439b69060df4bf482a01871fcb95
```

Comment:
Ok, so few extra notes:
1. 'June 7th' is no longer correct, also please use PV instead, don't make people think what the snapshot is named.
2. Please clean up the commit message. It's nice to list changes you did but there's really no point in having all this vcs-snapshot back-and-forth logged ;-). Also the 'fixup' thing.
3. Footer (Signed-off-by here) needs to be separated from text by empty line. Also, please don't use Signed-off-by since we didn't officially approve DCO yet.

Reply:
done